pg_search/src/postgres/customscan/pdbscan/mod.rs

```diff
@@ -1175,26 +1179,29 @@ fn compute_exec_which_fast_fields(
         )
     };
 
+    if exec_which_fast_fields.len() != builder.target_list().len() {
+        pgrx::log!(
+            "Expected to extract {} fast fields, but only found: {exec_which_fast_fields:?}. \
+             Falling back to Normal execution.",
+            builder.target_list().len()
+        );
+        return None;
+    }
```

Comment:
Discussed with @mdashti : this should really be _more_ than just length, and actually aware of the structure of the target list. For example, if we extract two fast fields from the first target list entry, and then none from the second target list entry, that is incorrect.

But that limitation already existed, so will add a column to TODO-NEW-ISSUE.